### DISCUSSION TITLE
onnx java invalid input shape

### DISCUSSION THREAD

**Discussion by waigel** [snakers4/silero-vad#320] [Q&A]
Hi, I am trying to use the VAD model with onnxruntime in Java.
Running into problems with the input shape: `Name:'Conv_606' Status Message: Invalid input shape: {4}`. 
Is there anyone who has a working implementation in Java or can help me find the implementation problem.

I created a gist with an example implementation: https://gist.github.com/waigel/c42cefa4bd327c62df45dee6685b1f36

**Comment by sreekj** [comment]
Were you able to solve this ?
